k8s/src/main/scala/io/buoyant/k8s/v1.scala

```diff
@@ -96,7 +96,7 @@ package object v1 {
       HighResTimer.Default,
       stats,
       RetryBudget.Infinite
-    )
+    ) andThen new LogFailureFilter(log, stats)
```

Comment:
My thinking was that without the infinite retry filter catching all failures, a failure should bubble up and blow up or log at a higher level.  But probably there's no logging at the higher level either so I think you're probably right we should log failure here regardless of if we are swallowing them or not.